src/constants/campus.data.ts

```diff
@@ -0,0 +1,36 @@
+import { Campus, CampusId } from "../types/main";
+
+const Campuses: Campus[] = [
+  {
+    id: CampusId.SGW,
+    description: "",
+    displayName: "Sir George Williams Campus",
+    region: {
+      latitude: 45.495376377001215,
+      longitude: -73.5777054168284,
+      latitudeDelta: 0.013275900094058102,
+      longitudeDelta: 0.009238533675670624
+    },
+    buildings: []
+  },
+  {
+    id: CampusId.Loyola,
+    displayName: "Loyola",
```

Comment:
minor but maybe change to "Loyola Campus" to keep it consistent

Reply:
Good point, I'll do it now real quick, but again the actual values don't matter as much as the structure.